README.md

```diff
@@ -170,6 +170,18 @@ lspconfig.tsserver.setup({
             -- inlay hints
             auto_inlay_hints = true,
             inlay_hints_highlight = "Comment",
+            inlay_hints_priority = 200, -- priority of the hint extmarks
+            inlay_hints_throttle = vim.o.updatetime, -- throttle the inlay hint request, set to 0 for no throttle
```

Comment:
Do you think there's a use case for exposing this separately? I think using `updatetime` is totally sensible, and I can't imagine many users would want to change it.

Reply:
I originally expose this option for users who want to disable the throttled behavior like the previous implementation, if they encounter any problems after upgrade. If the throttling works fine, we can certainly remove it.

Reply:
In that case I think it's better to test these changes a bit more before merging. I'd prefer not to expose an option that we'll remove in the future, since it would qualify as a breaking change.

Reply:
After looking at `:h updatetime` I think I misunderstood the usage of this option. The option defaults to a quite large value `4000` and is used by nvim internally to determine the frequency of calling `fsync()`. So it may be better to set the throttle time to a rather low value (200, etc.) and expose the option to let users modify it if they want tsserver to respond either faster or slower.

Reply:
I think we can also lift the option to plugin-level, for example, named as `delay` in order to be used by other module if possible.

Reply:
I see, I had thought that other plugins were also using `updatetime`, but that's because it also affects `CursorHold` (which isn't related to this). I'm always hesitant to add more options, but it's fine in this case (and I think we should keep it local to this specific functionality, since we never know how `tsserver` will behave in different environments).